xml/ldap_searchresult.xml

```diff
@@ -347,11 +347,14 @@
 
   <!-- End of Microsoft Windows Section -->
 
-  <fingerprint pattern="(?im:top\x04..penLDAProotDSE)">
+  <fingerprint pattern="(?im:\x04..penLDAProotDSE)">
     <description>OpenLDAP</description>
     <example _encoding="base64">
       dm9iamVjdENsYXNzMRYEA3RvcAQPT3BlbkxEQVByb290RFNFMA==
     </example>
+    <example _encoding="base64">
+      BBVzdHJ1Y3R1cmFsT2JqZWN0Q2xhc3MxEQQPT3BlbkxEQVByb290RFNFMBwEDQ
```

Comment:
I think this might be missing padding or similar; Looks like the the Go test suite fails:

```
time="2024-07-02T14:38:06Z" level=error msg="error verifying examples in recog-content/xml/ldap_searchresult.xml: base64: (?i-s:\\x04..PENLDAPROOTDSE): illegal base64 data at input byte 60 (BBVzdHJ1Y3R1cmFsT2JqZWN0Q2xhc3MxEQQPT3BlbkxEQVByb290RFNFMBwEDQ)"
```

Reply:
indeed. I updated the b64 data. Can you trigger the test again? Thank you